src/main/java/com/likelion/backendplus4/talkpick/batch/news/article/infrastructure/collector/config/executor/BatchJobExecutor.java

```diff
@@ -0,0 +1,78 @@
+package com.likelion.backendplus4.talkpick.batch.news.article.infrastructure.collector.config.executor;
+
+import org.quartz.DisallowConcurrentExecution;
+import org.quartz.JobExecutionContext;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.JobParametersInvalidException;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.batch.core.repository.JobExecutionAlreadyRunningException;
+import org.springframework.batch.core.repository.JobInstanceAlreadyCompleteException;
+import org.springframework.batch.core.repository.JobRestartException;
+import org.springframework.stereotype.Component;
+
+import com.likelion.backendplus4.talkpick.batch.news.article.exception.error.ArticleCollectorErrorCode;
+import com.likelion.backendplus4.talkpick.batch.news.article.exception.ArticleCollectorException;
+
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Quartz에 의해 트리거되는 Spring Batch Job 실행 클래스.
+ * JobLauncher를 통해 {@code rssJob}을 수동 실행하며, 매 실행 시 고유한 JobParameters를 생성하여 중복 실행을 방지한다.
+ *
+ * - @DisallowConcurrentExecution: 이전 실행이 끝나기 전에는 새로운 실행이 중첩되지 않도록 제한
+ * - JobParameters에 timestamp를 포함시켜 매번 다른 인스턴스로 실행되도록 설정
+ *
+ * 이 클래스는 단순한 실행자 역할만 하며, 실제 배치 로직은 {@code rssJob} 내부에 정의되어 있다.
+ *
+ * @since 2025-05-10
+ */
+@Component
+@DisallowConcurrentExecution
+@RequiredArgsConstructor
+public class BatchJobExecutor implements org.quartz.Job {
+	private final JobLauncher jobLauncher;
+	private final Job articleCollectorBatchJob;
+
+	/**
+	 * Quartz 트리거에 의해 호출되는 메서드.
+	 * 내부적으로 Spring Batch Job을 실행하는 로직을 위임한다.
+	 *
+	 * @param jobExecutionContext Quartz 실행 컨텍스트
+	 * @author 함예정
+	 * @since 2025-05-10
+	 */
+	@Override
+	public void execute(JobExecutionContext jobExecutionContext) {
+		startSpringBatchJob();
+	}
+
+	/**
+	 * Spring Batch Job을 JobLauncher를 통해 실행한다.
+	 * 각 실행마다 timestamp 파라미터를 부여하여 중복 실행 방지.
+	 * 예외 발생 시 {@link ArticleCollectorException}으로 변환하여 처리한다.
+	 *
+	 * @author 함예정
+	 * @since 2025-05-10
+	 */
+	private void startSpringBatchJob() {
+		JobParameters params = new JobParametersBuilder()
+			.addLong("timestamp", System.currentTimeMillis())
+			.toJobParameters();
+
+		try {
+			jobLauncher.run(articleCollectorBatchJob, params);
+		} catch (JobExecutionAlreadyRunningException e) {
+			throw new ArticleCollectorException(ArticleCollectorErrorCode.JOB_ALREADY_RUNNING);
+		} catch (JobRestartException e) {
+			throw new ArticleCollectorException(ArticleCollectorErrorCode.JOB_RESTART_FAIL);
+		} catch (JobInstanceAlreadyCompleteException e) {
+			throw new ArticleCollectorException(ArticleCollectorErrorCode.JOB_ALREADY_COMPLETE);
+		} catch (JobParametersInvalidException e) {
+			throw new ArticleCollectorException(ArticleCollectorErrorCode.INVALID_JOB_PARAMETER);
+		} catch (Exception e) {
+			throw new ArticleCollectorException(ArticleCollectorErrorCode.UNKNOWN_ERROR);
+		}
```

Comment:
P3:
개인적으로는,,, 이것들을 숨기는 쪽이 바람직하다고 생각해서 찾아보니 Exception을 판별하는 클래스를 빼는 방식으로 구현하는 방법도 있더라구요 아래에 코드를 첨부하니 의견을 들려주세요!
```suggestion
		try {
    jobLauncher.run(articleCollectorBatchJob, params);
} catch (Exception e) {
    ArticleCollectorErrorCode code = exceptionTranslator.translate(e);
    throw new ArticleCollectorException(code);
}

@Component
public class ExceptionTranslator {
    private static final Map<Class<? extends Exception>, ArticleCollectorErrorCode> CODE_MAP =
        Map.of(
            JobExecutionAlreadyRunningException.class,    ArticleCollectorErrorCode.JOB_ALREADY_RUNNING,
            JobRestartException.class,                   ArticleCollectorErrorCode.JOB_RESTART_FAIL,
            JobInstanceAlreadyCompleteException.class,   ArticleCollectorErrorCode.JOB_ALREADY_COMPLETE,
            JobParametersInvalidException.class,         ArticleCollectorErrorCode.INVALID_JOB_PARAMETER
        );

    public ArticleCollectorErrorCode translate(Exception e) {
        return CODE_MAP.getOrDefault(e.getClass(), ArticleCollectorErrorCode.UNKNOWN_ERROR);
    }
}
```